### PR TITLE
Fr - grammar.xml : False positive corrections in PLACE_DE_LA_VIRGULE rule

### PR DESCRIPTION
**Pull request by OswaldLuc**
French  - False positive corrections in file grammar.xml by adding antipattern in PLACE_DE_LA_VIRGULE rule group.

### CODE REVIEW

**Comment by danielnaber** [comment]
Thanks, but please remove the `<rule name="PLACE_DE_LA_VIRGULE[4]">` etc. - the number is added automatically by LT, and there will be confusion if it's in the `name` attribute.

**Comment by udomai** [comment]
…what you could do is leave it in as a comment, which is quite helpful in long rulegroups:

```xml
<rule><!-- PLACE_DE_LA_VIRGULE[1] -->
```

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -53893,8 +53937,9 @@ though, and has since been slightly modified)-->
                 <example>Peut-être l'<marker>apprécie-t-il aussi</marker>.</example>
                 <example>Les <marker>saviez-vous aussi</marker> agiles ?</example>
                 <example>On peut commencer <marker>par elle aussi</marker>.</example>
+                <example>C'est <marker> elle aussi</marker>qui habite le coeur des mères.</example>
```
Comment (dpelle) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:53940]: coeur → cœur

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -53820,8 +53853,12 @@ though, and has since been slightly modified)-->
                 <url>https://bescherelle.ca/virgules/</url>
                 <example correction=", te dit-elle,">Il le fait<marker> te dit-elle</marker> mais ce n'est pas vrai.</example>
                 <example correction=", leur dit-elle,">Il le pense<marker> leur dit-elle</marker> mais ce n'est pas vrai.</example>
+                <example correction=", leur dit-elle,">Il le pense<marker> leur dit-elle</marker> mais ce n'est pas vrai.</example>
+                <example>Et même, pendant notre maximum de vitesse, ne <marker> se permit-il</marker> pas de narguer la frégate en en faisant le tour !</example>
+                <example>Aussi<marker> se  tint-il</marker> prudemment entre deux eaux.</example>
```
Comment (dpelle) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:53858]: There should be only one space (not 2) between words: set tint-il

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 1 above)
Comment (dpelle) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:53940]: It's odd that there are 2 spaces and that the 2nd space is in between <marker>…</marker>

Comment (udomai) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:53940]: ALT + 0156 on Win machines 🤓

**Pull request closed without merging**